workspace-loader/package.json

```diff
@@ -23,7 +23,7 @@
     "typescript": "^2.7.1",
     "uglifyjs-webpack-plugin": "^1.1.8",
     "webpack": "^3.10.0",
-    "webpack-dev-server": "^2.11.1",
+    "webpack-dev-server": "^3.1.11",
```

Comment:
dev server v3.x is requiring a webpack 4.0 no ?
```
"peerDependencies": {
    "webpack": "^4.0.0"
  },
```

here we're using webpack 3

Reply:
Yes, it's correct, then I need to update webpack to 4.0 :(